### DISCUSSION TITLE
Persisting cache for local images processed with 11ty/image

### DISCUSSION THREAD

**Discussion by btrem** [11ty/eleventy#2619] [Q&A]
I'm using 11ty/image. Each time I run 11ty `--serve`, all of my images are reprocessed, making the initial build time very slow. According to https://www.11ty.dev/docs/plugins/image/#disk-cache, this shouldn't happen.

> Eleventy will skip processing files that are unchanged and already exist in the output directory.

Issue  11ty/eleventy-img#51 seems to indicate that persisting the cache to disk has been added to image as of 1.0.0. Is there something I need to do to my async image function to enable the disk cache feature?

**Comment by pdehaan** [comment]
Ah, sorry, I missed this earlier...

I haven't tried persistent cache stuff yet, but you might have better luck in the https://github.com/11ty/eleventy-img/issues repo.
Looks like the feature was added in https://github.com/11ty/eleventy-img/pull/116
I did find a reference to `useCache` in the PR, but looking through the source, it looks like it should be `true` by default. (at least if I'm understanding https://github.com/11ty/eleventy-img/blob/e51ad8e1da4a7e6528f3cc8f4b682972ba402a67/img.js#L57 correctly)
Also some interesting bits on in-memory cache at https://www.11ty.dev/docs/plugins/image/#in-memory-cache, notably this, which may or may not apply to you:

> _"This in-memory cache is maintained across builds in watch/serve mode. If you quit Eleventy, the in-memory cache will be lost."_

**Comment by btrem** [comment]
I'm a bit hesitant, because there's no discussion forum in 11ty/eleventy-img. I could open an issue, but I'm not sure it's a bug. Far more likely I've misunderstood something.
